acme/acme/client_test.py

```diff
@@ -741,6 +740,18 @@ def test_head_get_post_error_passthrough(self):
         self.assertRaises(requests.exceptions.RequestException,
                           self.net.post, 'uri', obj=self.obj)
 
+class ErrorTest(unittest.TestCase):
+    """Tests for urllib3 error conditions"""
+    # pylint: disable=too-many-instance-attributes,too-many-public-methods
+
+    def setUp(self):
+        self.response = mock.MagicMock(
+            ok=True, status_code=http_client.OK, headers={}, links={})
+        self.net = mock.MagicMock()
+        self.net.get.return_value = self.response
+
+    def test_urllib_error(self):
+         self.assertEqual(self.response, self.net.get("http://localhost:19123/nonexistent.txt"))
```

Comment:
What we want to do here is assert that this get raises an exception. Elsewhere in this file, you'll see `assertRaises`, but that's not quite what we want: we don't want to test just the type of the exception, but also its content. So set up a try/catch block around the `get` call, and in the catch block, verify that the raised exception has the correct message.